docs/getting-started/serving-for-inference-and-training.md

```diff
@@ -46,7 +55,7 @@ import featureform as ff
 
 client = ff.ServingClient(host)
 buffer_size = 1000
-for features, labels in client.training_set(name, variant).shuffle(buffer_size):
+for row in client.training_set(name, variant).shuffle(buffer_size):
```

Comment:
does the thing above not work? I think it would assuming its not batched

Reply:
lemme check

Reply:
`Traceback (most recent call last):
  File "../../../featureform/train.py", line 7, in <module>
    for feature, label in dataset:
TypeError: cannot unpack non-iterable Row object`